src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/custom_wl.py

```diff
@@ -0,0 +1,275 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import azure.cli.command_modules.backup.custom_help as custom_help
+# pylint: disable=import-error
+
+from knack.log import get_logger
+
+from azure.mgmt.recoveryservicesbackup.models import AzureVMAppContainerProtectionContainer, ProtectionContainerResource
+
+from azure.cli.core.util import CLIError, sdk_no_wait
+from azure.cli.command_modules.backup._client_factory import (
+    backup_protected_items_cf, backup_protection_containers_cf, protectable_containers_cf)
+
+logger = get_logger(__name__)
+
+fabric_name = "Azure"
+default_policy_name = "DefaultPolicy"
+os_windows = 'Windows'
```

Comment:
Please remove unused variables (line 19 to 24)

Reply:
Not all of them are unused. Fabric name is being used in almost every call. I will check for others.